pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -115,8 +115,10 @@ func (ds *dockerService) RunPodSandbox(config *runtimeapi.PodSandboxConfig) (str
 // after us?
 func (ds *dockerService) StopPodSandbox(podSandboxID string) error {
 	var namespace, name string
+	var checkpointErr, statusErr error
 	needNetworkTearDown := false
 
+	// Try to retrieve sandbox information from docker daemon or sandbox checkpoint
```

Comment:
It will try to retrieve sandbox checkpoint instead.

If both container and checkpoint does not exist (This actually happens when kubelet internal cache still has the sandbox but GC already clean it up.), then we only has sandbox ID. We can still proceed to teardown with only the ID, there is no network namespace either, then CNI bridge plugin will just try to do best effort clean up.